e2e/keywords/setting.resource

```diff
@@ -4,8 +4,24 @@ Documentation    Setting Keywords
 Library    ../libs/keywords/setting_keywords.py
 
 *** Keywords ***
-Set setting ${setting_key} to ${setting_value}
-    update_setting    ${setting_key}    ${setting_value}
-
 Set setting ${setting_key} to ${setting_value} will fail
-    Run Keyword And Expect Error    *    update_setting    ${setting_key}    ${setting_value}    retry=False
\ No newline at end of file
+    Run Keyword And Expect Error    *    update_setting    ${setting_key}    ${setting_value}    retry=False
+
+Setting ${setting_key} is set to ${setting_value}
```

Comment:
NIT: Do we need this renaming? All other functions use a Simple Present Tense verb as the 1st word.